README.md

```diff
@@ -6,20 +6,21 @@ Kanban capstone is a JavaScript group project employing Kanban board
 If you like what you see and have a project you need coded, don't hesitate to contact me.
 
 
+
 # Built With
 
 
 - Major languages: HTML, JavaScript, and CSS,
-- Technologies used: Web services, webpack, Jest for testing
+- Technologies used: Web services, webpack, Jest for testing, babelrc
 
 
 # Live Demo
 
-[Live Demo Link]().
+[Live Demo Link](https://emmy-github-webdev.github.io/kanban-capstone/dist/).
 
 # Presentation
 
-[Video]().
+[Video](https://www.loom.com/share/8912312a029f453398a27e478ff1c18c).
```

Comment:
- [ ] Great job explaining your project 👍 however according to the requirements, all the members that worked on the project should appear in the video and talk about the project. in order to do so, you can use the Zoom recording feature when you're on a call with your partner.